source/fundamentals/indexes.txt

```diff
@@ -397,6 +397,34 @@ Refer to the :manual:`Unique Indexes page </core/index-unique>` in the MongoDB s
 
 .. driver-content-end
 
+Clustered Indexes
+~~~~~~~~~~~~~~~~~
+
+**Clustered indexes** are indexes that improve the performance of
+insert, update, and delete operations on **clustered collections**.
+Clustered collections store documents ordered by the clustered index key
+value.
+
+To create a clustered index, specify a ``ClusteredIndexOptions`` in
+your ``CreateCollectionOptions``. The ``ClusteredIndexOptions`` must
+specify the ``_id`` field as the key and the unique field as ``true``.
+
+The following example creates a clustered index on the ``_id`` field in
+the ``tea.vendors`` collection:
```

Comment:
Issue: the collection name used in the example is "vendors" and not "tea.vendors".
I think you can actually have a collection name that includes a "." character, so it is not trivial to figure out.